data/default.html

```diff
@@ -60,13 +62,16 @@
      frame.onload = updatePlayer;
 
      function updatePlayer(ev) {
-       if (!frame.src) return;
```

Comment:
looks like this may have been [lost in the sauce](https://www.youtube.com/watch?v=HY5tIymnvWw)